.github/pull_request_template.md

```diff
@@ -8,7 +8,6 @@ Please fill in the checklist below and comment as needed.
 - [ ] Is this a feature addition? Briefly describe.
 - [ ] Did you modify QuaC-Watch config file? If so, did you modify multiqc template
   `configs/multiqc_config_template.jinja2` and script `src/quac_watch/create_mutliqc_configs.py` as necessary?
-- [ ] Did you perform system-level testing manually as described in master readme doc? Did it pass completely? If not why?
```

Comment:
Github actions now can take over system testing, but one thing it still doesn't test is whether quac works as expected sending jobs to slurm cluster. Would you call this an acceptable gap? I'm divided but leaning towards calling it as acceptable gap.

Reply:
In light of what we have seen as part of JOSS reviews, I now feel we should include a step to run a system testing job that utilizes sending jobs to slurm cluster.

Reply:
Hmmm that would be a bit more work but would truly be a full system test. Some of the issues noted have been due to the different environments set ups at different locations so this testing wouldn't alleviate that possibility. However, having a test her where jobs are submitted and handled through slurm would be more complete. I'm not sure though how much we gain from this kind of test though.

Reply:
To add to our earlier sidebar on this topic, the line of interest has been modified to this:

```
Did you perform system-level testing manually, using `--subtasks_slurm` flag, as described in the [documentation](https://quac.readthedocs.io/en/stable/system_testing/)? Did it pass completely? If not why?
```

Unsure why the changes made are not showing up in PR view above.